[catch2] build failure in Linux (WSL) in NTFS fs

**Host Environment**
- OS: Linux (Gentoo) in WSL
- Compiler: gcc (Gentoo 12.0.0_pre9999 p1, commit 5b1ce655b25040048861af6c0264cb667b66fcd7) 12.0.0 20210626 (experimental)

**To Reproduce**
Install Gentoo in WSL
Install vcpkg in NTFS filesystem
Mount the NTFS fs in Gentoo.

`./vcpkg install --triplet x64-linux catch2`

**Failure logs** 

```
-- Building x64-linux-rel
CMake Error at ports/catch2/portfile.cmake:19 (file):
  file RENAME failed to rename

    /mnt/g/Github/jessica/vcpkg/packages/catch2_x64-linux/share/Catch2

  to

    /mnt/g/Github/jessica/vcpkg/packages/catch2_x64-linux/share/catch2

  because: Directory not empty

Call Stack (most recent call first):
  scripts/ports.cmake:139 (include)
```

No additional failure logs.

--- a/ports/catch2/portfile.cmake
+++ b/ports/catch2/portfile.cmake
@@ -16,8 +16,10 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-file(RENAME "${CURRENT_PACKAGES_DIR}/share/Catch2" "${CURRENT_PACKAGES_DIR}/share/catch2")
-file(RENAME "${CURRENT_PACKAGES_DIR}/debug/share/Catch2" "${CURRENT_PACKAGES_DIR}/debug/share/catch2")
+file(RENAME "${CURRENT_PACKAGES_DIR}/share/Catch2" "${CURRENT_PACKAGES_DIR}/share/catch2_")
+file(RENAME "${CURRENT_PACKAGES_DIR}/share/catch2_" "${CURRENT_PACKAGES_DIR}/share/catch2")
+file(RENAME "${CURRENT_PACKAGES_DIR}/debug/share/Catch2" "${CURRENT_PACKAGES_DIR}/debug/share/catch2_")
+file(RENAME "${CURRENT_PACKAGES_DIR}/debug/share/catch2_" "${CURRENT_PACKAGES_DIR}/debug/share/catch2")
 
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/Catch2)
 

--- a/ports/catch2/vcpkg.json
+++ b/ports/catch2/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "catch2",
   "version-semver": "2.13.6",
+  "port-version": 1,
   "description": "A modern, header-only test framework for unit testing.",
   "homepage": "https://github.com/catchorg/Catch2"
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1146,7 +1146,7 @@
     },
     "catch2": {
       "baseline": "2.13.6",
-      "port-version": 0
+      "port-version": 1
     },
     "cccapstone": {
       "baseline": "9b4128ee1153e78288a1b5433e2c06a0d47a4c4e-1",

--- a/versions/c-/catch2.json
+++ b/versions/c-/catch2.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "bd73b1d5e8994fb0327c333cd77400577f05e31c",
+      "version-semver": "2.13.6",
+      "port-version": 1
+    },
+    {
       "git-tree": "17d502dbaa50c2e6d255331addb14259372fb6c5",
       "version-semver": "2.13.6",
       "port-version": 0

